tests/server.c

```diff
@@ -144,7 +144,7 @@ send_response(struct conndata *conn)
   struct rustls_connection *rconn = conn->rconn;
   const char *prefix = "HTTP/1.1 200 OK\r\nContent-Length:";
   const int body_size = 10000;
-  const int response_size = strlen(prefix) + 15 + body_size;
+  int response_size = strlen(prefix) + 15 + body_size;
```

Comment:
Why get rid of the const here? I think that is causing the test failures:

```
tests/server.c: In function ‘send_response’:
Makefile:40: recipe for target 'target/server.o' failed
tests/server.c:160:38: error: format ‘%ld’ expects argument of type ‘long int’, but argument 3 has type ‘int’ [-Werror=format=]
   fprintf(stderr, "strlen response %ld\n", response_size);
                                    ~~^
                                    %d
tests/server.c:166:8: error: comparison between signed and unsigned integer expressions [-Werror=sign-compare]
   if(n != response_size) {
        ^~
```

Note that we could also probably fix the tests by declaring response_size a size_t.

Reply:
> Why get rid of the const here? 

Because it's reassigned further down. And it better be `size_t response_size = ..`.
But regarding the `%ld` you're using. It better be `%zd` since that matches an `int` for 32-bit mode.